python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ipv6_router_ospf.yml

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  ipv6_router_ospf:
+    type: dict
+    display_name: IPv6 router OSPF Configuration
+    keys:
+      process_ids:
+        type: list
+        primary_key: id
+        unique_keys:
+          - vrf
```

Comment:
I am not sure this needs to be unique. I think it is possible to have multiple processes for a single VRF.

Reply:
More than 1 OSPFv3 instance is not supported
<img width="600" alt="Screenshot 2025-03-24 at 5 06 39 PM" src="https://github.com/user-attachments/assets/b1bd93fa-6050-400f-b8c7-e457bd3576d9" />

Reply:
ok cool. Thank you for checking.